meganav.yaml

```diff
@@ -1,3 +1,699 @@
+            - title: Kubernetes
+              description: Containerised workloads
+              url: /kubernetes
+            - title: Charmhub
+              description: Store for Juju operators
+              url: https://charmhub.io/
+            - title: Juju
+              description: Charmed Operator framework
+              url: https://juju.is/
+            - title: Ubuntu Certified Hardware
+              description: Machines you can trust
+              url: /certified
+
+    - title: Ubuntu OS
+      primary_links:
+        - title: Products
+          links:
+            - title: Ubuntu Desktop
+              description: Secure Enterprise Linux
+              url: /desktop
+            - title: Ubuntu Server
+              description: Ubuntu optimsed for servers
+              url: /server
+            - title: Ubuntu Core
+              description: Tiny embedded Linux
+              url: /core
+            - title: Ubuntu FIPS
+              description: Secure Linux for Government
+              url: /security/fips
+            - title: WSL2
+              description: Windows Subsystem for Linux
+              url: /wsl
+            - title: Multipass
+              description: VMs for Windows, macOS & Linux
+              url: https://multipass.run/
+      secondary_links:
+        - title: Links
+          links:
+            - title: Upgrade Ubuntu
+              url: /tutorials/upgrading-ubuntu-desktop#1-before-you-start
+            - title: Ubuntu dcoumentation
+              url: https://help.ubuntu.com/
+            - title: Desktop Install
+              url: /tutorials/install-ubuntu-desktop#1-overview
+            - title: Server install
+              url: /server/docs/installation
+            - title: Server Guide
+              url: /server/docs
+
+    - title: Private cloud
+      primary_links:
+        - title: Products
+          links:
+            - title: Ubuntu Server
+              description: Ubuntu optimsed for servers
+              url: /server
+            - title: Landscape
+              description: Monitor and manage servers
+              url: https://landscape.canonical.com/
+            - title: OpenStack
+              description: Private cloud provisioning
+              url: /openstack
+            - title: MicroStack
+              description: Small private cloud
+              url: https://microstack.run/
+            - title: MAAS
+              description: Bare metal provisioning
+              url: https://maas.io/
+            - title: Ceph
+              description: Open source software storage
+              url: /ceph
+            - title: Anbox cloud
+              description: Virtual Android devices on demand
+              url: https://anbox-cloud.io/
+            - title: Open Source MANO
+              description: Network Functions Virtualisation
+              url: /telco/osm
+            - title: JAAS
+              description: Juju as a Service
```

Comment:
```suggestion
              description: Juju as a service
```
Not sure on this one, but I don't think 'service' should be capitalised

Reply:
It's meant to be similar in format to "Software as a Service/SaaS" i.e. "JaaS":

![image](https://user-images.githubusercontent.com/2376968/196660952-9f02504b-f30d-40e6-9a44-fcfd73370a6d.png)